### PR TITLE
Add Flask sample application for Profiler.

### PR DESCRIPTION
**Pull request by mirelap-amazon**
*Issue #, if available:*

*Description of changes:*

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: sample-demo-flask-app/flaskr/blog.py [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+def _get_environment_variable(key, example_value):
+    value = os.getenv(key)
+    if value is None:
+        raise RuntimeError("Environment variable " + key + " must be set, e.g. " + example_value)
+    return value
+
+
+s3_key_name = 'resources/example-image.png'
+s3_bucket_name = _get_environment_variable(key="DEMO_APP_BUCKET_NAME",
+                                           example_value="test-images-for-my-demo-django-app")
```
Comment (gimki) [inline sample-demo-flask-app/flaskr/blog.py:25]: Flask?

File: sample-demo-flask-app/flaskr/schema.sql [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+);
+
+CREATE TABLE post (
+  id INTEGER PRIMARY KEY AUTOINCREMENT,
+  author_id INTEGER NOT NULL,
+  created TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  title TEXT NOT NULL,
+  body TEXT NOT NULL,
+  FOREIGN KEY (author_id) REFERENCES user (id)
+);
```
Comment (gimki) [inline sample-demo-flask-app/flaskr/schema.sql:17]: Optional: Add a new line

File: sample-demo-flask-app/generate-traffic.sh [hunk 3]
```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+START_DATETIME=$(date +%s)
+
+COUNT=1000
+
+for i in $(seq 1 $COUNT); do
+  echo "> Running #$i from ${COUNT}... "
+  sleep 1
+  pytest integration_tests
```
Comment (gimki) [inline sample-demo-flask-app/generate-traffic.sh:10]: Shouldnt curl be sufficient to generate traffic? What is running the integration test's work here?

Comment (mirelap-amazon) [reply sample-demo-flask-app/generate-traffic.sh:10]: I'm trying to diversify the traffic, that's why I added something as integration test, to have a get, post, more frames.

File: sample-demo-flask-app/flaskr/blog.py (hunk 1 above)
Comment (mirelap-amazon) [reply sample-demo-flask-app/flaskr/blog.py:25]: Yes, will update, thanks.

File: sample-demo-flask-app/flaskr/schema.sql (hunk 2 above)
Comment (mirelap-amazon) [reply sample-demo-flask-app/flaskr/schema.sql:17]: Yes, will update, thanks.

File: sample-demo-flask-app/README.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+The main entry point is in `sample-demo-flask-app/flaskr/__init__.py` that initializes applications, like the main one from `sample-demo-flask-app/flaskr/blog.py`.
+
+* The configuration for gunicorn is in `gunicorn.py`.
+
+* There in the `gunicorn.py` you can find the configuration for CodeGuru Profiler to start and to log.
+
+### Run the service.
+
+Start the service using the gunicorn configuration.
+    ```bash
```
Comment (gimki) [inline sample-demo-flask-app/README.md:55]: Unnecessary indent?

File: sample-demo-flask-app/requirements.txt [hunk 5]
```diff
@@ -0,0 +1,4 @@
+flask==1.1.2
+requests
+codeguru_profiler_agent
+boto3
```
Comment (gimki) [inline sample-demo-flask-app/requirements.txt:4]: gunicorn too?

**Review by gimki — APPROVED** [review]

**Pull request merged**